nodejs/BUILD

```diff
@@ -1,107 +1,33 @@
-load("@container_structure_test//:defs.bzl", "container_structure_test")
-load("@rules_oci//oci:defs.bzl", "oci_image", "oci_image_index")
-load("@rules_pkg//:pkg.bzl", "pkg_tar")
-load("//:distro.bzl", "DISTROS")
-load("//nodejs:node_arch.bzl", "node_arch")
+load(":nodejs.bzl", "nodejs_image", "nodejs_image_index")
+load(":node_arch.bzl", "node_arch")
+load("//:distro.bzl", "DISTROS", "PREVIEW_DISTROS")
 
 package(default_visibility = ["//visibility:public"])
 
 NODEJS_MAJOR_VERSIONS = ("20", "22", "24")
 
-MODE = [
-    "",
-    "_debug",
-]
+NODEJS_DISTROS = DISTROS + PREVIEW_DISTROS
 
-USER = [
-    "root",
-    "nonroot",
-]
+# NODEJS_ARCHITECTURES = ARCHITECTURES
+# using node_arch for now, but we can probably get rid of that
```

Comment:
Once we drop support for 20 and 22, we can drop node_arch in favor of `BASE_ARCHITECTURES + ["s390x", "ppc64le"]` (similar to ARCHITECTURES, just without `arm`)